### DISCUSSION TITLE
Welcome to the GitHub ROCm Community

### DISCUSSION THREAD

**Discussion by saadrahim** [ROCm/ROCm#1832] [General]
AMD is enabling the GitHub discussions as a trial to encourage users to discuss ROCm concepts here. We will be redirecting discussions that are using the GitHub Issues to the community here. Please use GitHub Issues to report issues only. You may request new discussion threads using this category. Community involvement is encouraged. Have fun and please be respectful of each other.

**Comment by Ezrealfuck** [comment]
6750gre use what rocm??

**Reply by Ezrealfuck** [reply]
I don't konw use the version of rocm ,my device is 6750gre
